DOCS/man/options.rst

```diff
@@ -6825,8 +6825,11 @@ them.
     files contain uncompressed LUTs. Their size depends on the
     ``--icc-3dlut-size``, and can be very big.
 
-    NOTE: On ``--vo=gpu``, this is not cleaned automatically, so old, unused
-    cache files may stick around indefinitely.
+    On `--vo=gpu-next`, files that have not been accessed in the last 24 hours
+    may be cleared if the cache limit (1.5 GiB) is exceeded.
```

Comment:
this is awfully lot for just a video player IMO
(yes, I know this is icc)

Reply:
I understand that, but it won't happen in normal configurations. We in fact support 512x512x512 3dlut size, so the cache should accommodate that. In normal operation is is almost not possible to saturate that. Nevertheless, I don't want to limit the cache size and the limit is in fact only the safety net to not blow up in case of something weird happens and stop at this 1.5GiB which is indeed high for a player, but still not like 10GiB...
```
--icc-3dlut-size=<auto|RxGxB> [...] Sizes may range from 2 to 512.
```

Suggestions welcome, I don't want to make it more complex by adding configurable cache size, but in the same time I want to support all supported use-cases. Even if the use-case is completely stupid it is on user, not on us.

Reply:
Well ok then.